dist/scss/wds-components/_toggles.scss

```diff
@@ -0,0 +1,76 @@
+@import '../wds-variables/colors';
+
+.wds-toggle {
+	&__input {
+		display: none;
+	}
+
+	&__label {
+		cursor: pointer;
+		font-size: $wds-typescale-size-base;
+		padding: 7px 5px 7px 52px;
+		position: relative;
+
+		&:empty {
+			padding-left: 45px;
+		}
+
+		&::before, &::after {
+			content: '';
+			margin: 0;
+			outline: 0;
+			position: absolute;
+			top: 50%;
+			transform: translateY(-50%);
+			transition: all .3s ease;
+		}
+
+		&::before {
+			background-color: $wds-color-light-c5;
+			border-radius: 8px;
+			height: 14px;
+			left: 6px;
+			width: 34px;
+		}
+
+		&::after {
+			background-color: $wds-color-light-c2;
+			border-radius: 50%;
+			box-shadow: 0 3px 1px -2px rgba(0, 0, 0, .14), 0 2px 2px 0 rgba(0, 0, 0, .098), 0 1px 5px 0 rgba(0, 0, 0, .084);
+			height: 20px;
+			left: 5px;
+			width: 20px;
```

Comment:
It's funny but you can use `square` mixin to draw a circle :)
https://github.com/Wikia/design-system/blob/master/style-guide/styles/wds-mixins/_square.scss